src/main/java/com/impactupgrade/nucleus/service/segment/SfdcCrmService.java

```diff
@@ -140,6 +140,24 @@ public void addContactToCampaign(CrmContact crmContact, String campaignId) throw
     sfdcClient.insert(campaignMember);
   }
 
+  @Override
```

Comment:
I don't believe TER has the smsOptIn field defined in their environment, and doesn't have a related field in SFDC as far as I'm aware... But other than that I think this should work fine.

Reply:
Rather, most Orgs don't have that environment property set, but it exists in nucleus-engine's default-environment.json. Point still stands about TER and SFDC though